base_classes/NXprocess.nxdl.xml

```diff
@@ -45,12 +50,22 @@
     </field>
     <group type="NXnote">
         <doc>
-            The note will contain information about how the data was processed
-            or anything about the data provenance. 
-            The contents of the note can be anything that the processing code 
-            can understand, or simple text.
+            The note will contain information about how the data was
+            processed or anything about the data provenance. The
+            contents of the note can be anything that the processing
```

Comment:
"the processing code", you mean the program that is will be written under ``program`` if so might be good to mak a reference in the docstring to this ``program`` concept ?